src/main/java/racingcar/Car.java

```diff
@@ -10,14 +12,19 @@ public Car(String name) {
     }
 
     public Car(String name, Integer position) {
-        if (name == null || name.isBlank() || name.length() > 5) {
-            throw new IllegalArgumentException("자동차 이름은 5 글자를 넘을 수 없다.");
-        }
+        checkValidCarName(name);
 
         this.name = name;
         this.position = position;
     }
 
+    private static void checkValidCarName(String name) {
+        if (name == null || name.isBlank() || name.length() > LIMIT_CAR_NAME_LENGTH) {
+            throw new IllegalArgumentException(
+                    String.format("자동차 이름은 %d 글자를 넘을 수 없다.", LIMIT_CAR_NAME_LENGTH));
+        }
```

Comment:
```suggestion
        if (name == null || name.isBlank()) {
            throw new IllegalArgumentException("자동차 이름은 비어 있을수 없습니다.");
        }
        
        if (name.length() > LIMIT_CAR_NAME_LENGTH) {
            throw new IllegalArgumentException(
                    String.format("자동차 이름은 %d 글자를 넘을 수 없다.", LIMIT_CAR_NAME_LENGTH));
        }
        
```

유효성 검사는 위와 같이 분리하는게 좋아보입니다 😄